protocol/v1/qbft/instance/partial_change_round.go

```diff
@@ -30,6 +31,8 @@ func (i *Instance) uponChangeRoundPartialQuorum() pipelines.SignedMessagePipelin
 			i.ResetRoundTimer()
 			//i.ProcessStageChange(qbft.RoundStateChangeRound)
 
+			i.State().ProposalAcceptedForCurrentRound.Store((*specqbft.SignedMessage)(nil))
```

Comment:
why reseting when bump round?

Reply:
@nivBlox It's according to spec: https://github.com/bloxapp/ssv-spec/blob/main/qbft/round_change.go#L72
However, [there's a to-do ](https://github.com/bloxapp/ssv-spec/blob/main/qbft/round_change.go#L71) to check if we actually need to do that, so I'm not sure if we need to keep that or not.